_posts/2020-10-16-status-update.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "2020-10-16: IAC Team Status Update"
+author: DavidS
+categories:
+  - team
+  - status
+tags:
+---
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-docker#666`][puppetlabs-docker-pr-666]: Thanks to [optiz0r][optiz0r]
```

Comment:
@david22swan can you fill out the thank you messages below?

Reply:
I've added the PR titles now. anything specific you'd like to see?